sdk/scenario-format/src/interpret_trait.rs

```diff
@@ -1,13 +1,23 @@
 use std::path::PathBuf;
 
-#[derive(Default)]
+use crate::value_interpreter::VMIdentifier;
+
+#[derive(Default, Clone)]
 pub struct InterpreterContext {
     pub context_path: PathBuf,
+    pub vm_type: VMIdentifier,
```

Comment:
If we added this field, we should probably also use it when interpreting addresses.

Just so we don't forget: search for the function where addresses are being built, and place these 2 bytes on positions 9 and 10 in the address.

Can be done in separate PR.

Reply:
this was already dome when I added the VM bytes where we build the sc address